src/routes/profile.svelte

```diff
@@ -54,7 +72,7 @@
   }
 
   function changeUsername({ detail }) {
-    api.patch('/account/telegram', {
+    api.patch(`/account/${account.email}/telegram`, {
```

Comment:
Shouldn't it be `/accounts` instead of `/account`? Same thing for subsequent changes

Reply:
Fixed in a commit I cannot see here for some reason